src/Show.jl

```diff
@@ -326,22 +342,36 @@ q - 1
 
 ```
 """
-function nrchars(t::CharTable, char::Int64)
-	if char > irrchartypes(t)
-		throw(DomainError(char, "Cannot calculate number of characters in reducible types."))
-	else
-		o=t.ring(1)
-		result=t[char].sum(o//o)
-		return shrink(result)
-	end
-end
-function nrchars(t::SimpleCharTable, char::Int64)
+function nrchars(t::Table, char::Int64)
 	if char > chartypes(t)
 		throw(DomainError(char, "Character type is out of range."))
```

Comment:
Is this even needed? Won't `t[char]` throw an error anyway? Perhaps a less nice one -- but if we want a nicer one, I think we should put it into `Base.getindex(ct::CharTable, i::Integer)`, and remove the check here.

Reply:
No, it is not needed. But we do have this in other functions too. See for example `printinfochar`. Shouldn't we remove it there as well?

Reply:
Ok, I'll do that in a follow up PR where I also remove these checks for the class types.